server/src/main/java/org/opentosca/toscana/plugins/cloudfoundry/FileCreator.java

```diff
@@ -43,6 +45,8 @@
     public static final String APPLICATION_FOLDER = "app";
     public static String deploy_name = "application";
 
+    private final static Logger logger = LoggerFactory.getLogger(FileCreator.class);
```

Comment:
Is there a reason why you don't use our custom log mechanism?
Without it, the user won't see the logs at the cli / webapp.

In order to implement it, simply inject the `Log` instance contained in the 
`TransformationContext` instance to your class (via constructor injection).
Then, simply call:
```java
this.logger = log.getLogger(getClass());
```